compass/landice/tests/ismip6_run_ais/projection/streams.landice

```diff
@@ -0,0 +1,132 @@
+<streams>
+
+<immutable_stream name="input"
+                  type="input"
+                  filename_template="{{ input_file_ais }}"
+                  input_interval="initial_only"/>
+
+<immutable_stream name="restart"
+                  type="input;output"
+                  filename_template="rst.$Y.nc"
+                  filename_interval="output_interval"
+                  output_interval="0100-00-00_00:00:00"
+                  reference_time="1995-01-01_00:00:00"
+                  clobber_mode="truncate"
+                  precision="double"
+                  input_interval="initial_only"/>
+
+<stream name="regionsInput"
+        type="input"
+        mode="forward;analysis"
+        filename_template="regionMasks.nc"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="regionCellMasks"/>
+
+</stream>
+
+<stream name="ismip6"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_TF_forcing }}"
+    input_interval="0001-00-00_00:00:00"
+    record_interval="0001-00-00_00:00:00"
+                  reference_time="1950-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_3dThermalForcing"/>
+        <var name="ismip6shelfMelt_zOcean"/>
+</stream>
+
+<stream name="ismip6_2"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_basal_param }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_deltaT"/>
+        <var name="ismip6shelfMelt_basin"/>
+        <var name="ismip6shelfMelt_gamma0"/>
+</stream>
+
+<stream name="ismip6_3"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_shelf_melt_offset }}"
+        input_interval="initial_only"
+        runtime_format="single_file">
+        <var name="ismip6shelfMelt_offset"/>
+</stream>
+
+<stream name="ismip6_smb"
+        type="input"
+        mode="forward;analysis"
+        filename_template="{{ file_ais_SMB_forcing }}"
+    input_interval="0001-00-00_00:00:00"
+    record_interval="0001-00-00_00:00:00"
+                  reference_time="1950-01-01_00:00:00"
+        runtime_format="single_file">
+        <var name="sfcMassBal"/>
+</stream>
+
+<stream name="output"
```

Comment:
We'll need to look through the fields required to be submitted to ISMIP6 and figure out what fields and frequency we will need.  We will likely need one file output annually for state fields and then a second file being output every timestep for flux fields that we can time average annually in post-processing.

Reply:
I've edited the `streams.landice.template` file in [https://github.com/hollyhan/compass/commit/57f85d80c8ebf3c1b6551608d071e54e41576b58](url) by going through the required ismip6 outputs [https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.3.3_Table_A1:_Variable_request_for_ISMIP6](url). There are a number of variables on the table that I need to check/confirm with you which variables on registry they correspond to, and which variables are calculated in post-processing steps rather than outputted in streams.